### PR TITLE
Add descriptions on how to build from source & Add wasi-stub building script

### PR DESCRIPTION
**Pull request by yy-zhong**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 set -euxo pipefail
+# Download the wasm-minimal-protocol repo and build wasi-stub tool from source.
+(
```
Comment (Enter-tainer) [inline build.sh]: i'm not really familar with shell script. will this run every time when you run ./install.sh?

**Review by Enter-tainer — APPROVED** [review]
thank you!

File: build.sh (hunk 1 above)
Comment (Enter-tainer) [reply build.sh]: I'd suggest removing this part and move this to README if i understand it correctly

Comment (yy-zhong) [reply build.sh]: The quoted part does run everytime you run the `install.sh` script. 

The highlighted code could be moved into the README. Or, if I may suggest, create an option in the shell script, allowing users to choose whether they wish to build the `wasi-stub` tool from source.

Hope you can share your thoughts! 😉

Comment (Enter-tainer) [reply build.sh]: I prefer moving them to readme. In my experience i don't like random shell script installing things to my system --- I mean I prefer reading the code and copy-paste them to terminal.

Comment (yy-zhong) [reply build.sh]: Sure thing! Just made those changes in the latest commit in this pull request 😉.

**Review by Enter-tainer — APPROVED** [review]
Thank you!

**Pull request merged**